From 57d98d6142d490332839ec261a2a4d01c31de4a4 Mon Sep 17 00:00:00 2001
From: Andy Wingo <wingo@pobox.com>
Date: Fri, 26 Mar 2010 13:20:00 +0100
Subject: remove spurious declarations in c-tokenize.lex

* libguile/c-tokenize.lex: Remove spurious declarations of flex-internal
  functions, as their prototypes are changing in upstream flex.

Origin: http://git.savannah.gnu.org/cgit/guile.git/commit/?id=83a7b43bf2bf445af9d2611252deaacb4de7095e
Added-by: Rob Browning <rlb@defaultvalue.org>
---
 THANKS                  |  2 ++
 libguile/c-tokenize.lex | 14 --------------
 2 files changed, 2 insertions(+), 14 deletions(-)

diff --git a/THANKS b/THANKS
index 48a105a..0a3e84e 100644
--- a/THANKS
+++ b/THANKS
@@ -74,6 +74,7 @@ For fixes or providing information which led to a fix:
             Dan McMahill
           Roger Mc Murtrie
           Scott McPeak
+         Andrew Milkowski
             Tim Mooney
         Han-Wen Nienhuys
             Jan Nieuwenhuizen
@@ -86,6 +87,7 @@ For fixes or providing information which led to a fix:
            Arno Peters
 	    Ron Peterson
           David Pirotte
+         Sergey Poznyakoff
             Ken Raeburn
         Andreas Rottmann
            Hugh Sasse
diff --git a/libguile/c-tokenize.lex b/libguile/c-tokenize.lex
index 938a5d2..dc72257 100644
--- a/libguile/c-tokenize.lex
+++ b/libguile/c-tokenize.lex
@@ -24,20 +24,6 @@ INTQUAL		(l|L|ll|LL|lL|Ll|u|U)
    an error for that. */
 #define YY_NO_INPUT
 
-int yylex(void);
-
-int yyget_lineno (void);
-FILE *yyget_in (void);
-FILE *yyget_out (void);
-int yyget_leng (void);
-char *yyget_text (void);
-void yyset_lineno (int line_number);
-void yyset_in (FILE * in_str);
-void yyset_out (FILE * out_str);
-int yyget_debug (void);
-void yyset_debug (int  bdebug);
-int yylex_destroy (void);
- 
 int filter_snarfage = 0;
 int print = 1; 
 
